kafka/consume.go

```diff
@@ -73,29 +75,83 @@ func NewConsumer(log logrus.FieldLogger, conf Config, opts ...ConfigOpt) (*Confl
 		return nil, err
 	}
 
-	confluenceConsumer := &ConfluentConsumer{
-		c:    consumer,
-		conf: conf,
-		log:  log,
+	if conf.Timeout == 0 {
+		conf.Timeout = DefaultTimout
 	}
 
-	return confluenceConsumer, nil
+	part := int32(-1)
```

Comment:
The config is already a pointer. Don't understand why we need to change the type in here instead of use the same in both (pointer to).